_posts/2021-09-29-solidity-0.8.9-release-announcement.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.9 Release Announcement'
+date: '2021-09-29'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.9](https://github.com/ethereum/solidity/releases/tag/v0.8.9) fixes two important bugs:
+
+ - one related to [immutables of signed integer types shorter than 256 bits](https://blog.soliditylang.org/2021/09/29/signed-immutables-bug/) introduced in Solidity 0.6.5
```

Comment:
```suggestion
 - a bug related to [immutables of signed integer types shorter than 256 bits](https://blog.soliditylang.org/2021/09/29/signed-immutables-bug/), through which sign extension (cleanup) of its value is not always properly performed, introduced in Solidity 0.6.5
```

Would consider adding short info for people to have context that don't want to read the bug posts.